metpy/plots/_util.py

```diff
@@ -15,10 +15,11 @@
 from ..units import concatenate
 
 
-def add_timestamp(ax, time=None, x=0.99, y=-0.04, ha='right', high_contrast=False, **kwargs):
-    """Add a timestamp at plot creation time.
+def add_timestamp(ax, time=None, x=0.99, y=-0.04, ha='right', high_contrast=False,
+                  pretext='Created: ', time_format='%Y-%m-%dT%H:%M:%SZ', **kwargs):
+    """Add a timestamp to a plot.
 
-    Adds an ISO format timestamp with the time of plot creation to the plot.
+    Adds a timestamp to a plot, defaulting to the time of plot creation.
```

Comment:
Mention ISO format as the default?

Reply:
In the main docstring? Don't really mention defaults in others do we?

Reply:
```
metpy/calc/thermo.py
202:        The maximum number of iterations to use in calculation, defaults to 50.
204:        The desired relative error in the calculated value, defaults to 1e-5.
512:        for air. Defaults to the ratio for water vapor to dry air
632:        for air. Defaults to the ratio for water vapor to dry air.
668:        for air. Defaults to the ratio for water vapor to dry air.
703:        for air. Defaults to the ratio for water vapor to dry air.
779:        Psychrometer coefficient. Defaults to 6.21e-4 K^-1.
1152:        Defaults to using the bottom pressure or height.
1201:        The axis corresponding to the vertical in the temperature array, defaults to 0.
1204:        Defaults to False.
1206:        The maximum number of iterations to use in calculation, defaults to 50.
1208:        The desired absolute error in the calculated value, defaults to 1e-6.
1617:        for air. Defaults to the ratio for water vapor to dry air.
1620:        The bottom of the layer in pressure. Defaults to the first observation.
1622:        The depth of the layer in hPa. Defaults to the full profile if bottom is not given,
1692:        The bottom of the layer in pressure. Defaults to the first observation.
1694:        The depth of the layer in hPa. Defaults to the full profile if bottom is not given,
1735:        The axis corresponding to vertical in the potential temperature array, defaults to 0.
1776:        The axis corresponding to vertical in the potential temperature array, defaults to 0.
1814:        The axis corresponding to vertical in the potential temperature array, defaults to 0.
```

Reply:
You can put it in the parameter documentation if you prefer.